integrations/axum/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![forbid(unsafe_code)]
-
+#![feature(once_cell)]
```

Comment:
This errors on nightly (with a message that `once_cell` is now stable on nightly 1.71) but also on stable (because it's enabling a nightly feature)... I think we should just keep the `once_cell` dependency until this is stable on stable.

Reply:
Sure, I'll remove that